facade/facade.go

```diff
@@ -80,7 +80,8 @@ type Facade struct {
 	hostRegistry  auth.HostExpirationRegistryInterface
 	deployments   *PendingDeploymentMgr
 
-	isvcsPath string
+	isvcsPath              string
+	serviceRunLevelTimeout time.Duration
```

Comment:
_No change requested_, but this parameter doesn't fit in the facade from a logical perspective.  To me that's an indication that the shutdown sequencing code does not belong in the facade, as we discussed earlier.